beakerx/beakerx/tabledisplay/tabledisplay.py

```diff
@@ -91,7 +93,10 @@ def convertType(self, object_type, value):
         if object_type == "float64":
             return "double"
         if object_type == "int64":
-            return "integer"
+            if value > (pow(2, 31) - 1) or value < -(pow(2, 31)):
```

Comment:
use sys.maxint etc, do not compute these values.

Reply:
unclear to me that it's the right value at all.  js supports 56 bits before we have to switch.